packages/xdl/src/start/startDevServerAsync.ts

```diff
@@ -66,10 +66,15 @@ export async function startDevServerAsync(
     options.maxWorkers = startOptions.maxWorkers;
   }
 
-  const { server, middleware, messageSocket } = await runMetroDevServerAsync(projectRoot, options);
   // TODO: reduce getConfig calls
   const projectConfig = getConfig(projectRoot, { skipSDKVersionRequirement: true });
 
+  // Use the unversioned metro config.
+  options.unversioned =
```

Comment:
want to make it possible to opt in with env var also?

Reply:
i guess opting in with metro.config.js manually is fine too

Reply:
maybe we'll add an env var in the future to make testing expo-cli easier, but for now this is fine.